WooCommerce/Classes/ViewRelated/Coupons/CouponDetails/CouponDetailsViewModel.swift

```diff
@@ -34,6 +34,10 @@ final class CouponDetailsViewModel: ObservableObject {
     ///
     @Published private(set) var hasWCAnalyticsDisabled: Bool = false
 
+    /// Indicates whether a network call is in progress
```

Comment:
It could be better to indicate here that `isLoading` will be used just for the `delete` action? Because I noticed that when the view is loading data, `isLoading` will never change, so it might be confusing. Maybe renaming the variable to `isDeletionLoading` or `isDeletionInProgress` could help too.

Reply:
Good point, updated in 08236b0689417a428d5eb18fd2da976f472726c0.